src/store/plugins/load-geojson-style-and-data.plugin.js

```diff
@@ -19,11 +21,50 @@ async function load(url) {
     }
 }
 
+async function autoReloadData(store, geoJsonLayer) {
+    if (!geoJsonLayer.updateDelay) {
+        return
+    }
+    // clearing any pre-existing interval for this layer
+    if (intervalsByLayerId[geoJsonLayer.id]) {
+        clearInterval(intervalsByLayerId[geoJsonLayer.id])
+    }
+    log.debug(
+        `Adding auto-reload of data for layer ${geoJsonLayer.id} according to its updateDelay`,
+        geoJsonLayer
+    )
+    // creating the new interval to reload this layer's data
+    intervalsByLayerId[geoJsonLayer.id] = setInterval(async () => {
+        try {
+            const { data } = await load(geoJsonLayer.geoJsonUrl)
+            const layerCopy = geoJsonLayer.clone()
+            layerCopy.geoJsonData = data
+            // we update through the action updateLayers, so that if multiple copies of the same GeoJSON layer are present,
+            // they will all be updated with the fresh data
+            store.dispatch('updateLayers', {
+                layers: [layerCopy],
+                ...dispatcher,
+            })
```

Comment:
Code is all good, but just as information, here instead of doing the clone you do 
```
store.dispatch('updateLayers', {layers: [{id: geoJsonLayer.id, geoJsonData: data}], ...dispatcher})
```

But no need to change both are ok and good in my opinion